engine/runtime/src/main/java/org/enso/interpreter/runtime/data/EnsoFile.java

```diff
@@ -243,6 +242,13 @@ public String getName() {
     return this.truffleFile.getName();
   }
 
+  @Builtin.Method(name = "size")
+  @Builtin.WrapException(from = IOException.class)
+  @CompilerDirectives.TruffleBoundary
+  public long getSize() throws IOException {
+    return this.truffleFile.size();
+  }
```

Comment:
are you sure this also gets us the `or total size of a folder.`, I'm not sure looking at the Truffle implementation.

Reply:
Its what it appeared to do in Windows. As I don't need that feature I will make it error for directories.

Reply:
I'm not sure, maybe it was working correctly, I just wanted to make sure we test it :)
but we can disallow for now, that works as well